account_asset_management/migrations/8.0.3/pre-migration.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# Copyright 2009-2017 Noviat
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.DEBUG)
+
+column_renames = {
+    'account_account': [
+        ('asset_category_id', 'asset_profile_id', 'Asset Profile'),
+    ],
+    'account_asset': [
+        ('asset_value', 'depreciation_base', 'Depreciation Base'),
+        ('category_id', 'profile_id', 'Asset Profile'),
+    ],
+    'account_invoice_line': [
+        ('asset_category_id', 'asset_profile_id', 'Asset Profile'),
+    ],
+    'account_move_line': [
+        ('asset_category_id', 'asset_profile_id', 'Asset Profile'),
+    ],
+}
+
+table_renames = [
+    ('account_asset_asset', 'account_asset'),
+    ('account_asset_category', 'account_asset_profile'),
+    ('account_asset_depreciation_line', 'account_asset_line')]
+
+model_renames = [
+    ('account.asset.asset', 'account.asset'),
+    ('account.asset.category', 'account.asset.profile'),
+    ('account.asset.depreciation.line', 'account.asset.line')]
+
+view_refs = [
+    'account_asset_management.view_account_asset_form',
+    'account_asset_management.view_invoice_asset_category',
+    'account_asset_management.view_account_invoice_asset_form',
+    'account_asset_management.view_account_move_line_form_inherit',
+    'account_asset_management.view_account_move_asset_form'
+]
+
+
+def is_module_installed(cr, module):
+    """ Check if `module` is installed.
+    :return: True / False
+    """
+    cr.execute(
+        "SELECT id FROM ir_module_module "
+        "WHERE name=%s and state IN ('installed', 'to upgrade')", (module,))
+    return bool(cr.fetchone())
+
+
+def table_exists(cr, table):
+    """ Check whether a certain table or view exists """
+    cr.execute('SELECT 1 FROM pg_class WHERE relname = %s', (table,))
+    return cr.fetchone()
+
+
+def rename_columns(cr, column_spec):
+    for table in column_spec.keys():
```

Comment:
Please use openupgradelib instead of copying these methods (without attribution :-O)